docs/team/samuelfoo.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Again, I don't think this file should be in this PR at all. It may be causing the merge conflict.
This file and the changed AboutUs.md seems to belong in the "Add Samuel's Details" (PR #8) pull request instead, but they appear in this branch.

Reply:
Thanks. Removed samuelfoo.md.